rfcs/profiler.md

```diff
@@ -0,0 +1,146 @@
+# Tech Spec: Performance Evaluation & Monitoring for Single Cell
```

Comment:
The spec branding is SOMA. The impl branding is TileDB-SOMA. There is no "Single Cell" branding in our terminology. Please don't use this.